@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -231,6 +231,14 @@ export class TileGeometryCreator {
         this.createTextElements(tile, decodedTile, textFilter);
 
         this.createLabelRejectionElements(tile, decodedTile);
+
+        // HARP-7899, disable ground plane for globe
+        if (tile.dataSource.addGroundPlane && tile.projection.type === ProjectionType.Planar) {
+            // The ground plane is required for when we change the zoom back and we fall back to the
+            // parent, in that case we reduce the renderOrder of the parent tile and this ground place
+            // ensures that parent doesn't come through.
+            TileGeometryCreator.instance.addGroundPlane(tile, -10000);
```

Comment:
Can we avoid magic number?  Since it is related to fallback maybe use instead:
``
-MapView.FALLBACK_RENDER_ORDER_OFFSET / 2 
``

Reply:
Done, and added comment.